src/integTest/resources/sample.servlet/testCreateLibertyConfigDir.gradle

```diff
@@ -0,0 +1,107 @@
+group = 'liberty.gradle'
+version = '1'
+
+buildscript {
+    repositories {
+        mavenCentral()
+		mavenLocal()
+        maven {
+            name = 'Sonatype Nexus Snapshots'
+            url = 'https://oss.sonatype.org/content/repositories/snapshots/'
+        }
+    }
+    dependencies {
+        classpath 'net.wasdev.wlp.gradle.plugins:liberty-gradle-plugin:1.1-SNAPSHOT'
+        classpath files('gradle/wlp-anttasks.jar')
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'war'
+apply plugin: 'liberty'
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+compileJava.options.encoding = 'UTF-8'
+
+ext {
+    // Liberty server properties
+    wlpServerName = 'LibertyProjectServer'
+    serverDirectory = "${project.buildDir}/wlp/usr/servers/${wlpServerName}"
+    testServerHttpPort = 9080
+    testServerHttpsPort = 9443
+
+    // This is set in the ibm-web-ext.xml file
+    warContext = 'myLibertyApp'
+    appName = 'LibertyProject'
+    packageFile = "${project.buildDir}/${appName}.zip"
+    packagingType = 'usr'
+    
+}
+
+liberty {
+    install {
+        runtimeUrl = "http://repo1.maven.org/maven2/com/ibm/websphere/appserver/runtime/wlp-webProfile7/17.0.0.2/wlp-webProfile7-17.0.0.2.zip"
+    }
+    serverName = wlpServerName
+    
+    configDirectory = "${project.projectDir}/src/main/liberty/config"
+    
+    packageLiberty {
+        archive = packageFile
+        include = packagingType
+    }
+}
+
+repositories {
+    mavenCentral()
+    maven {
+        name 'liberty-starter-maven-repo'
+        url 'http://liberty-starter.wasdev.developer.ibm.com/start/api/v1/repo'
+    }
+}
+
+dependencies {
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.apache.cxf:cxf-rt-rs-client:3.1.1'
+    testCompile 'org.glassfish:javax.json:1.0.4'
+    providedCompile group: 'javax.servlet', name: 'javax.servlet-api', version:'3.1.0'
+    runtime 'net.wasdev.wlp.starters.rest:runtime-pom:0.0.3'
+}
+
+task publishWar(type: Copy) {
+    from(war)
+    into("${serverDirectory}/dropins")
+}
+
+test {
+	println 'inside the test block'
+    reports.html.destination = file("$buildDir/reports/unit")
+    reports.junitXml.destination = file("$buildDir/test-results/unit")
+    exclude '**/it/**'
+}
+
+
+task integrationTest(type: Test) {
+    group 'Verification'
+    description 'Runs the integration tests.'
+    reports.html.destination = file("$buildDir/reports/it")
+    reports.junitXml.destination = file("$buildDir/test-results/it")
+    include '**/it/**'
+    exclude '**/unit/**'
+
+    systemProperties = ['liberty.test.port': testServerHttpPort, 'war.name': warContext]
+}
+
+task printMessageAboutRunningServer {
+    doLast {
+        println "The server is now running at http://localhost:${testServerHttpPort}/${warContext}"
+        println "To stop the server run 'gradle libertyStop'"
+    }
+}
+
+check.dependsOn 'integrationTest'
+libertyCreate.dependsOn 'installLiberty'
+integrationTest.dependsOn 'libertyCreate'
```

Comment:
I tried to go through one file in detail, but the comments apply to all.  Since this test is just about testing if the the configuration is copied correctly, let's remove anything related to the sample servlet, the war, etc.  You would only want that if the configuration you are copying is required to get the application started correctly and you were testing the application start which you are not.